rendezvous-server/src/handlers_to0.rs

```diff
@@ -69,27 +69,6 @@ pub(super) async fn ownersign(
         .into());
     }
 
-    // Now check the OV first public key: is it one we trust?
-    let manufacturer_pubkey = to0d
-        .ownership_voucher()
-        .header()
-        .manufacturer_public_key()
-        .clone();
-    log::trace!(
-        "Checking whether manufacturer key {:?} is trusted",
-        manufacturer_pubkey
-    );
-    if let Some(trusted_manufacturer_keys) = &user_data.trusted_manufacturer_keys {
-        if !trusted_manufacturer_keys.contains_publickey(&manufacturer_pubkey) {
-            return Err(Error::new(
-                ErrorCode::InvalidOwnershipVoucher,
-                messages::v11::to0::OwnerSign::message_type(),
-                "Ownership voucher manufacturer not trusted",
-            )
-            .into());
-        }
-    }
-
```

Comment:
This part is actually OK and we shouldn't delete it.
`to0d.OV.header[4]` holds the manufacturer's public key, as specified at <https://fidoalliance.org/specs/FDO/FIDO-Device-Onboard-PS-v1.1-20220419/FIDO-Device-Onboard-PS-v1.1-20220419.html#OwnershipVoucher>.
The manufacturer server stores its public cert there ([code](https://github.com/fdo-rs/fido-device-onboard-rs/blob/main/manufacturing-server/src/handlers/di.rs#L133)), so in this part we are just doing an allowlist check.

Reply:
this also comes [from this](https://fidoalliance.org/specs/FDO/FIDO-Device-Onboard-PS-v1.1-20220419/FIDO-Device-Onboard-PS-v1.1-20220419.html#TO0:~:text=It%20is%20preferred,such%20a%20trust.) :
> It is preferred that the Rendezvous Server has a basis on which to trust at least one public key within the Ownership Voucher. For example, the manufacturer who ran the DI protocol to configure the Device, thereby choosing the Rendezvous Server, may register public keys with the Rendezvous Server to establish such a trust.